examples/charm++/load_balancing/kNeighbor/kNeighbor.C

```diff
@@ -265,14 +339,17 @@ class Block: public CBase_Block {
 
       numNborsRcvd = 0;
       /* 1: pick a work size and do some computation */
-      int N = (thisIndex * thisIndex / num_chares) * 100;
+      //TODO
+      int N = (thisIndex * thisIndex / num_chares) * compute_work * (rand()%compute_var + 1) + l_factor*l_frequency;
```

Comment:
It's hard to figure out what this does, a comment and some renaming of variable would help (namely `compute_work` and `compute_var` are too similar).